src/main/resources/application.yml

```diff
@@ -72,6 +72,8 @@ metrics:
 ---
 spring.profiles: local
 cache.expiry_sec: 300
+cache.min_expiry: 60
```

Comment:
Since YAML supports hierarchical representation of parameters same could be expressed (better idiomatically) as follows:
```
cache:
  min_expiry: 60
  max_expiry: 28800
```